.bazelversion

```diff
@@ -1 +1 @@
-2.0.0
+2.1.1
```

Comment:
is this intentional?

Reply:
no; removed. I just now adjusted my dev flow so hopefully I don't commit this again.